### ISSUE TITLE
Should caddy2 be default branch?

### ISSUE THREAD

**Issue by Lingxi-Li** [caddyserver/forwardproxy#100]
Looking into the commits, thought `caddy2` should be the default branch or be merged to `master`. The current default `master` is outdated and confuses new comers.

**Comment by mholt** [comment]
Agreed it's confusing. See https://github.com/caddyserver/forwardproxy/pull/74 for the history. It will be merged into `master` as soon as it's done. (It's not done yet. It works, but not 100% with the same guarantees.)

**Comment by francislavoie** [comment]
@mholt this can be closed now, this was done.

**Comment by mholt** [comment]
True. Thanks @francislavoie and @gaby!
